Assets/MixedRealityToolkit/Providers/BaseDeviceManager.cs

```diff
@@ -14,15 +15,21 @@ namespace Microsoft.MixedReality.Toolkit.Core.Providers
     /// <summary>
     /// Base Device manager to inherit from.
     /// </summary>
+    // todo: inherit BaseDataProvider...
     public class BaseDeviceManager : BaseExtensionService, IMixedRealityDeviceManager
```

Comment:
we need to stop adding arbitrary todos to the repo. If this has another associated issue it needs to be logged here. Otherwise, if this is never done we really generate confusing content in the code.

Reply:
In principle I agree but am less anxious about todos included in part [x] of an incomplete series of pull requests.

Reply:
if this goes into mrtk_development it shouldn't be considered an incomplete series. Every time someone forks mrtk_development they'll have to pick through these.

Reply:
granted i agree that helpers will eventually be removed. I'd just prefer if you called out this service registration issue in the todos (some formal todo tag thats searchable) so we know what they're for/so a deep search can be done on the last pull request to make sure we've addressed everything.